### DISCUSSION TITLE
FramebufCharacter

### DISCUSSION THREAD

**Discussion by honboy** [micropython/micropython#12459] [ESP32]
Can the framebuf characters be added or modified to a size of 6*8? Because the 6*8 shape will not look crowded and the aspect ratio is more coordinated.thanks

**Comment by robert-hh** [comment]
If you ask for a 6x8 font instead of 5x8, do you want to fill all 6 columns or do you ask for one column space between the characters?

**Comment by honboy** [comment]
Leave a column of space between characters,Such characters look more beauty and the boundaries between characters are clearer.

**Comment by robert-hh** [comment]
Using e.g. the font-to-py library of @peterhinch https://github.com/peterhinch/micropython-font-to-py with the writer class you can use a font of your choice and size. These fonts looks much better.
